spec/fixtures/files/mira_xml.xml

```diff
@@ -21,15 +21,16 @@ http://www.openarchives.org/OAI/2.0/OAI-PMH.xsd">
           <tufts:filename>pdf-sample.pdf</tufts:filename>
           <tufts:visibility>open</tufts:visibility>
           <tufts:filename>3.pdf</tufts:filename>
+          <tufts:displays_in>dl</tufts:displays_in>
           <model:hasModel>Pdf</model:hasModel>
           <dc:title>President Jean Mayer speaking
           at commencement, 1987</dc:title>
           <dc11:description>
-            A long       description             with 
+            A long       description             with
                            plenty
           of          whitespace</dc11:description>
           <dc:abstract>
-            Another long       description             with 
```

Comment:
I think this file is intended to have trailing whitespace, and the corresponding test in spec/lib/tufts/import_record_spec.rb, is meant to explicitly test that it is retained.

I know this is a pain, but I think we need to keep it. https://twitter.com/no_reply/status/940737286446051329